src/integrations/browser.js

```diff
@@ -0,0 +1,56 @@
+import GaToSplit from './ga/GaToSplit';
+import SplitToGa from './ga/SplitToGa';
+import { GA_TO_SPLIT, SPLIT_TO_GA, SPLIT_IMPRESSION, SPLIT_EVENT } from '../utils/constants';
+
+/**
+ * Factory function for browser IntegrationsManager.
+ * The integrations manager instantiates integration modules, and bypass tracked events and impressions to them. 
+ * 
+ * @param {Context} context SplitFactory context
+ * 
+ * @returns integration manager or null if `integrations` are not present in settings.
+ */
+const integrationsManagerFactory = context => {
+  const settings = context.get(context.constants.SETTINGS);
+
+  // If no integrations settings, not return a integration manager
+  if (!settings.integrations)
+    return;
+
+  const listeners = [];
+
+  for (const integrationOptions of settings.integrations) {
+    const { type } = integrationOptions;
```

Comment:
These two lines become this for es5 build in npm (people that bundles) as well as our UMD bundle:

```
    for (var _iterator = _babel_runtime_corejs3_core_js_get_iterator__WEBPACK_IMPORTED_MODULE_0___default()(settings.integrations), _step; !(_iteratorNormalCompletion = (_step = _iterator.next()).done); _iteratorNormalCompletion = true) {
      var integrationOptions = _step.value;
      var type = integrationOptions.type;
```

Why not use a regular for loop? Or a forEach?
```
for (let i=0; i < settings.integrations.length; i++) {
  const integrationOptions = settings.integrations[i];
  const type = integrationOptions.type;
}

settings.integrations.forEach(integrationOptions => {
   const type = integrationOptions.type;
  // ... function body
}
```